MSAL/src/native_auth/network/errors/sign_up/MSALNativeAuthSignUpStartResponseError.swift

```diff
@@ -49,7 +49,7 @@ struct MSALNativeAuthSignUpStartResponseError: MSALNativeAuthResponseError {
 
 extension MSALNativeAuthSignUpStartResponseError {
 
-    func toSignUpStartPasswordPublicError() -> SignUpPasswordStartError {
+    func toSignUpStartPasswordPublicError() -> SignUpStartError {
```

Comment:
Do you think this method and `toSignUpStartPublicError()` can be merged? Given that we're merging both signUp methods, we don't have any more a signUp password public error.

Reply:
Methods merged